packages/store-sdk/test/analytics/useAnalyticsEvent.test.ts

```diff
@@ -0,0 +1,51 @@
+import { renderHook } from '@testing-library/react-hooks'
+
+import { useAnalyticsEvent } from '../../src/analytics/useAnalyticsEvent'
+import { wrap } from '../../src/analytics/wrap'
+import type { AddToCartEvent } from '../../src/analytics/events/add_to_cart'
+
+const eventSample: AddToCartEvent = {
+  type: 'add_to_cart',
+  data: {
+    items: [{ item_id: 'PRODUCT_ID' }],
+  },
+}
```

Comment:
I noticed this piece of code is in 3 different files, could we put it in a single one and import it in the others?

Reply:
Will do!